beacon_node/store/src/hot_cold_store.rs

```diff
@@ -43,6 +43,8 @@ use types::data_column_sidecar::{ColumnIndex, DataColumnSidecar, DataColumnSidec
 use types::*;
 use zstd::{Decoder, Encoder};
 
+const HISTORICAL_BLOB_BATCH_SIZE: usize = 1000;
```

Comment:
I think lets get rid of the batch size so that the entire Vec provided by the caller either commits or doesn't